include/net/downloader.h

```diff
@@ -195,6 +196,11 @@ struct downloader_host_cfg {
 	 * This string is used in case you are requesting a proxied file from a CoAP server.
 	 */
 	const char *proxy_uri;
+	/**
+	 * Maximum number of HTTP redirects.
+	 * Use 0 to set default value of CONFIG_DOWNLOADER_MAX_REDIRECTS.
```

Comment:
This seems incorrect. The implementation sets the value of `CONFIG_DOWNLOADER_MAX_REDIRECTS`, not its default.

Reply:
Ok, I see your point. I thought of the value of `CONFIG_DOWNLOADER_MAX_REDIRECTS` as the default, but that is not the default value of `CONFIG_DOWNLOADER_MAX_REDIRECTS`. I'll rewrite.